go.mod

```diff
@@ -3,22 +3,31 @@ module github.com/brave/nitriding
 go 1.19
 
 require (
-	github.com/brave/viproxy v0.1.2
+	github.com/containers/gvisor-tap-vsock v0.4.0
 	github.com/go-chi/chi/v5 v5.0.7
 	github.com/hf/nitrite v0.0.0-20211104000856-f9e0dcc73703
-	github.com/hf/nsm v0.0.0-20211106132757-1ae65a6a69ae
-	github.com/mdlayher/vsock v1.1.1
+	github.com/hf/nsm v0.0.0-20220930140112-cd181bd646b9
 	github.com/milosgajdos/tenus v0.0.3
-	golang.org/x/crypto v0.0.0-20220829220503-c86fa9a7ed90
-	golang.org/x/sys v0.0.0-20220825204002-c680a09ffe64
+	github.com/songgao/packets v0.0.0-20160404182456-549a10cd4091
+	github.com/songgao/water v0.0.0-20200317203138-2b4b6d7c09d8
+	github.com/vishvananda/netlink v1.2.1-beta.2
+	golang.org/x/crypto v0.3.0
```

Comment:
Go's tooling made this change because the Go folks finally started using tags for packages in golang.org/x/ (see https://github.com/golang/go/issues/21324). In fact, I just updated to the latest dependencies in beed0b9, which bumps the x/crypto package to v0.5.0.

Reply:
Cool! good to know this is the direction they're heading to prevent supply chain attacks through redeploying a new package under the same version